sitedata/config.yml

```diff
@@ -1,23 +1,20 @@
-name: MiniConf
-tagline: MINICONF <br> CONFERENCE <br> MANAGEMENT
-date: May 2020
-proceedings_title: Proceedings of the First MiniCon Conference
+name: ACL2020
+tagline: 2020 Annual Conference of <br> the Association for Computational Linguistics
```

Comment:
I'm a little confused -- whether we are "Annual Conference" or "Annual Meeting"?